tests/rbd/rbd_utils.py

```diff
@@ -773,6 +773,35 @@ def set_config(self, level, entity, key, value, **kw):
         """
         return self.exec_cmd(cmd=f"rbd config {level} set {entity} {key} {value}", **kw)
 
+    def set_ceph_config(self, entity, key, value, **kw):
```

Comment:
[optional] we can think of modifying the existing set_config and get_config method, according to the parameter passed in function.

Reply:
i feel it's good to keep `ceph` commands separate over `rbd` commands hence added this method.

Reply:
I agree with Sunil here, let's not combine rbd config and ceph config.

Reply:
But @sunilangadi2 set_ceph_config and get_ceph_config are generic methods not related to RBD object, so they can be moved out of the RBD class.

Reply:
Done